src/Handler/Workspace/Drop/DropWorkspaceHandler.php

```diff
@@ -9,15 +9,22 @@
 use Google\Service\CloudResourceManager\GetIamPolicyRequest;
 use Google\Service\CloudResourceManager\Policy;
 use Google\Service\CloudResourceManager\SetIamPolicyRequest;
+use Google\Service\Iam\Resource\ProjectsServiceAccounts;
 use Keboola\StorageDriver\BigQuery\CredentialsHelper;
 use Keboola\StorageDriver\BigQuery\GCPClientManager;
 use Keboola\StorageDriver\BigQuery\IAmPermissions;
 use Keboola\StorageDriver\Command\Workspace\DropWorkspaceCommand;
 use Keboola\StorageDriver\Contract\Driver\Command\DriverCommandHandlerInterface;
 use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+use Retry\BackOff\ExponentialBackOffPolicy;
+use Retry\Policy\CallableRetryPolicy;
+use Retry\RetryProxy;
+use Throwable;
 
 final class DropWorkspaceHandler implements DriverCommandHandlerInterface
 {
+    private const ERROR_CODES_FOR_RETRY = [401, 403, 429];
```

Comment:
https://cloud.google.com/datastore/docs/concepts/errors

RESOURCE_EXHAUSTED - [Verify that you did not exceed your project quota](https://cloud.google.com/datastore/docs/pricing#locating_quota_usage_information_for_your_app). If you exceeded a project quota, do not retry without fixing the problem.

Otherwise, retry with exponential backoff.

Tim ze tu len mazeme resources tak nenarazime na projektovu quotu ale mozeme narazit na rete limit, v response nieje nejaky cas ktory by sa dal pouzit tak som tam dal ten exponential backoff ako sa pise v dokumentacii